packages/marko-web-identity-x/README.md

```diff
@@ -0,0 +1,143 @@
+# BaseCMS Marko Wrapper for IdentityX
+
+## Installation
+
+1. Include `@base-cms/marko-web-identity-x` as a project/website dependency.
+
+2. Include IdentityX tenant configuration within your site
+```js
+// your-site/config/identity-x.js
+const IdentityX = require('@base-cms/marko-web-identity-x/config');
+
+const config = new IdentityX('<MY-APPLICATION-ID>');
+module.exports = config;
+```
+
+3. Include the IdentityX router **before all other routes!**
+```js
+// your-site/server/routes/index.js
+const IdentityX = require('./identity-x');
+
+module.exports = (app) => {
+  IdentityX(app);
```

Comment:
It looks like the route instructions are including twice

Reply:
They're not, I'm instructing the user to create an `./identity-x.js` route and include it in their `index.js` route. It's just named the same, but if anyone actually sets this up, they're just going to copy+paste anyway :P

Reply:
Updated docs to be more clear